modules/apps/headless-apio/media-object/media-object-apio-impl/src/main/java/com/liferay/media/object/apio/internal/architect/resource/MediaObjectNestedCollectionResource.java

```diff
@@ -113,8 +120,8 @@ public String getName() {
 		).addLinkedModel(
 			"creator", PersonIdentifier.class, FileEntry::getUserId
 		).addNestedList(
-			"encoding", this::_getMediaQueries,
-			this::_getMediaQueryNestedRepresentor
+			"adaptedMedia", this::_getAdaptiveMedias,
```

Comment:
mmm looking at the dictionary, media is already a plural form :S

Reply:
i'll change it again 😂